src/tm-threads.c

```diff
@@ -1797,58 +1797,72 @@ void TmThreadContinue(ThreadVars *tv)
     return;
 }
 
-/**
- * \brief Waits for all threads to be in a running state
- *
- * \retval TM_ECODE_OK if all are running or error if a thread failed
- */
-TmEcode TmThreadWaitOnThreadRunning(void)
+static TmEcode WaitOnThreadsRunningByClass(const int c)
```

Comment:
nit: else where these are thread types, not class.

Reply:
agree